reopt/Main_reopt.hs

```diff
@@ -189,8 +189,10 @@ data Args = Args
   -- ^ Path to llvm-mc
   --
   -- Only used when generating object file from assembly generated by llc.
+  , addEntryPoints :: ![String]
+  -- ^ List of extra entry points to start discovery from.
```

Comment:
Perhaps worth saying how this differs from `includeAddrs`, which (as I understand) is subtly different.